File: ui/src/test/java/org/wcl/mfn/ui/utilities/TestApplication.java
package org.wcl.mfn.ui.utilities;

import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.web.servlet.config.annotation.EnableWebMvc;
import org.springframework.boot.SpringApplication;

@SpringBootApplication(scanBasePackages = "org.wcl.mfn")
@EnableWebMvc
public class TestApplication {

    public static void main(String... args) {
        SpringApplication.run(TestApplication.class, args);
    }
}

File: ui/src/test/kotlin/org/wcl/mfn/ui/utilities/TestApplication.kt
package org.wcl.mfn.ui.utilities

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.web.servlet.config.annotation.EnableWebMvc
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["org.wcl.mfn"])
@EnableWebMvc
open class TestApplication
    fun main(args: Array<String>) {
        runApplication<TestApplication>(*args)
    }